src/components/ui/PriceDisplay.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class PriceDisplay extends React.Component {
+    static propTypes = {
+        fish: PropTypes.object.isRequired,
+        professionMultiplier: PropTypes.number.isRequired,
+    };
+
+    profMult = price => (Math.floor(price * this.props.professionMultiplier));
```

Comment:
Final nitpick - you don't need parentheses around the function body if it's on a single line.

`profMult = price => Math.floor(price * this.props.professionMultiplier);`